2023-03-27 18:12:41,468:logging_decorator:ERROR:Exception in fetch_blog: You are not allowed to view blog with id='private-blog'.                                
['Traceback (most recent call last):\n  File "/Users/benkaan/Desktop/projects/api_exception_handling/version3/logging_decorator.py"', ' line 81', ' in wrapper_log_exceptions\n    return func(*args']
2023-03-27 18:12:49,032:logging_decorator:ERROR:Exception in fetch_blog: You are not allowed to view blog with id='private-blog'.                                
['Traceback (most recent call last):\n  File "/Users/benkaan/Desktop/projects/api_exception_handling/version3/logging_decorator.py"', ' line 81', ' in wrapper_log_exceptions\n    return func(*args']
2023-03-27 18:12:51,040:logging_decorator:ERROR:Exception in fetch_blog: Unable to find blog with id='wrong-blog'                                
['Traceback (most recent call last):\n  File "/Users/benkaan/Desktop/projects/api_exception_handling/version3/logging_decorator.py"', ' line 81', ' in wrapper_log_exceptions\n    return func(*args']
